### DISCUSSION TITLE
Enable Inlay type hints

### DISCUSSION THREAD

**Discussion by KittyBorgX** [helix-editor/helix#6121] [Q&A] [answered]
Is there any way to enable inlay type hints for rust to make something like this work?
![image](https://user-images.githubusercontent.com/82328083/221590787-7a03ab0b-17e7-4ccb-afde-ae0e2fc63f00.png)
I'm referring to the `&Value` `B` and `IsNaN` in this image

**Comment by gabydd** [comment]
You can see https://github.com/helix-editor/helix/issues/2070 it's being worked on in the pr linked to the issue

**Reply by KittyBorgX** [reply]
Got it! Since it's implemented for golang, is there any way I can test it out locally? Or do I need to wait for the PR to be merged?

**Reply by gabydd** [reply]
Yep you can clone the pr and follow the install from source instructions or development instructions to try it out

**Reply by gabydd** [reply]
It can also be used for languages other then golang, rust included

**Comment by David-Else** [comment, marked as answer]
This is the correct link for the current type hints, and there are instructions how to test it: https://github.com/helix-editor/helix/pull/5934
